### PR TITLE
AO3-4195 Request more invites page required fields not sufficiently marked

### PR DESCRIPTION
**Pull request by awilson739**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added tests for any changed functionality?
* [x] Have you added the [JIRA](https://otwarchive.atlassian.net) issue number as the *first* thing in your pull request title (eg: `AO3-1234 Fix thing`)
* [x] Have you updated or commented on the JIRA issue with the information below?

## Issue

https://otwarchive.atlassian.net/browse/AO3-4195

## Purpose

What does this PR do? 
Adds the "* Required Information" notice at the top of the request invites form and adds (*) next to each label.

Note: I realize the notice is a bit off from the form. Perhaps that can be fixed after the first initial pass. 

## Testing Instructions

1. Log in as a user
2. My Dashboards
3. Click on "Invitations" button 
4. Click on "Request Invitations" button 
## References

Are there any other relevant issues / PRs / mailing lists discussions related to this? None that I know of

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?* Angela is just fine

(If you have an account on JIRA, please include the same name in the "Full name" field on your JIRA profile,
so we can find you and assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*
She
(If you do not fill this in, we will use your GitHub account name and will use "they" to avoid making assumptions about your gender.)

### CODE REVIEW

File: app/views/user_invite_requests/new.html.erb [hunk 1]
```diff
@@ -12,11 +12,12 @@
 
 <!--main content-->
 <%= form_for(@user_invite_request) do |f| %>
+<p class="required notice"><%= ts("* Required information") %></p>
   <%= error_messages_for @user_invite_request %>
   <dl>
-    <dt class="required"><%= f.label ts("How many invitations would you like? (max %{max})", :max => ArchiveConfig.MAX_USER_INVITE_REQUEST.to_s) %></dt>
+    <dt class="required"><%= f.label ts("How many invitations would you like? (max %{max})*", :max => ArchiveConfig.MAX_USER_INVITE_REQUEST.to_s) %></dt>
```
Comment (sarken) [inline app/views/user_invite_requests/new.html.erb]: When we put asterisks on labels, we like to keep them outside the `ts()` to make sure they can't get lost during the translation process, like so: `<%= f.label ts("How many invitations would you like? (max %{max})", :max => ArchiveConfig.MAX_USER_INVITE_REQUEST.to_s) + "*" %>`

File: app/views/user_invite_requests/new.html.erb [hunk 2]
```diff
@@ -12,11 +12,12 @@
 
 <!--main content-->
 <%= form_for(@user_invite_request) do |f| %>
+<p class="required notice"><%= ts("* Required information") %></p>
```
[the remark this answers is not in the document]
Comment (awilson739) [reply app/views/user_invite_requests/new.html.erb]: Fixed with new commit.

File: app/views/user_invite_requests/new.html.erb (hunk 1 above)
Comment (awilson739) [reply app/views/user_invite_requests/new.html.erb]: Fixed with new commit.

**Comment by awilson739** [comment]
I'll comment on the ticket :)

**Pull request merged**